rasa/shared/core/trackers.py

```diff
@@ -581,16 +582,26 @@ def as_story(self, include_source: bool = False) -> "Story":
         )
         return Story.from_events(self.applied_events(), story_name)
 
-    def export_stories(self, e2e: bool = False, include_source: bool = False) -> Text:
+    def export_stories(
+        self, e2e: bool = False, include_source: bool = False, story_format: str = ""
```

Comment:
As mentioned in the comment above, I think we can pass a writer here instead of `story_format`, and it shouldn't have a default value.